mobile/cmp/input/Select.js

```diff
@@ -105,6 +116,9 @@ Select.propTypes = {
     /** Text to display when control is empty. */
     placeholder: PT.string,
 
+    /** Text to display in header when in fullscreen mode. */
```

Comment:
nit:  might just call this 'title' and mention that it is currently used in `fullScreen` mode